samples/web/send-from-aspnetcore-webapi/sample.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Using NServiceBus in an ASP.NET Core WebAPI Application
+component: Core
+reviewed: 2016-09-27
+related:
+- nservicebus/hosting
+---
+
+This sample shows how to send messages to a NServiceBus endpoint from a ASP.NET Core WebAPI application. It only works for ASP.NET Core solutions targeting the full .NET Framework. The .NET Core runtime is currently not supported.
+
+Run the solution, a new browser window/tab opens, as well as a console application.
+
+The browser will open up the URL `http://localhost:51863/api/sendmessage`. An async [WebAPI](https://www.asp.net/web-api) controller handles the request. It creates an NServiceBus message and sends it to the endpoint running in the console application. The message has been processed successfully when the console application prints "Message received at endpoint". 
```

Comment:
As mentioned before, I think it would be more elegant to have a button on that page and send when the button is pressed.